src/model/encoding/__tests__/__snapshots__/convertFromHTMLToContentBlocks-test.js.snap

```diff
@@ -407,19 +590,6 @@ exports[`does not convert deeply nested html blocks when experimentalTreeDataSup
 
 exports[`does not convert deeply nested html blocks when experimentalTreeDataSupport is disabled 2`] = `
 Array [
-  Object {
```

Comment:
this demonstrates that this bug was already reproducible with the old non tree code path.